app.js

```diff
@@ -298,7 +300,14 @@ const render = () => {
 		event.id = index
 	})
 
-	event_list.innerHTML = Mustache.render(TEMPLATE, {data: visible_events}, {partial : PARTIAL_TEMPLATE});
+	const event = visible_events.find(event =>
+		new Date(event.date.end || event.date.start) >= new Date()
+	) ?? visible_events[visible_events.length - 1]
```

Comment:
Ked si nastavim filtre tak aby som nemal nic zobrazene tak toto bude undefined a potom na 307 to hodi error :)

Reply:
Jo to by sa asi patrilo fixnúť...

Reply:
fixed

Reply:
fixed